newIDE/app/src/UI/Alert/ConfirmDeleteDialog.js

```diff
@@ -15,9 +15,11 @@ type Props = {|
   title: MessageDescriptor,
   message: MessageDescriptor,
   fieldMessage: MessageDescriptor,
-  confirmText: string,
+  confirmHintText: string,
   onConfirm: () => void,
   onDismiss: () => void,
+  confirmText?: MessageDescriptor,
+  dismissText?: MessageDescriptor,
 |};
 
 function ConfirmDeleteDialog(props: Props) {
```

Comment:
canConfirm must use confirmHintText. I'm not sure confirmHintText is a great wording. I think it should stay confirmText and the labels should be `confirmButtonLabel` and `DismissButtonLabel`

Reply:
Yeah you're probably right, I'll update that

Reply:
done!